doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,8 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added a Kconfig option that configures the number of HID subscribers in :ref:`nrf_desktop_hid_forward` (:ref:`CONFIG_DESKTOP_HID_FORWARD_SUBSCRIBER_COUNT <config_desktop_app_options>`).
```

Comment:
Remove the changelog entry from this PR and add it as a comment to Divya's https://github.com/nrfconnect/sdk-nrf/pull/18192.